server/tests/integration/test_validate.py

```diff
@@ -197,9 +197,9 @@ async def test_validate_fusion(
     """Perform some basic tests on the fusion validation endpoint."""
     await check_validated_fusion_response(async_client, alk_fusion, "ALK fusion")
     await check_validated_fusion_response(async_client, ewsr1_fusion, "EWSR1 fusion")
-    await check_validated_fusion_response(
-        async_client, ewsr1_fusion_fill_types, "EWSR1 fusion needing type inference"
-    )
+    # await check_validated_fusion_response(
```

Comment:
I would like some opinions/eyes on this. I think because of the change I made in https://github.com/cancervariants/fusor/pull/174, we can't do inference to the extent of before. Is this a problem? Is it okay to leave for now and make an issue for improvement later?

Reply:
Looking at this now

Reply:
How often does this happen in the UI where a user is going through the form and no type is provided?

Reply:
I think we can save this for a new issue IMO. I think we'd just need to add a model_validator to `AssayedFusion` for this?

Reply:
it's currently not possible to get to this state without a provided fusion type. The UI always adds the type in the request before calling the validation function